apps/registry/pages/api/[payload].js

```diff
@@ -77,7 +81,7 @@ export default async function handler(req, res) {
       `https://api.github.com/users/${username}/gists?per_page=100`,
       {
         headers: {
-          Authorization: 'Bearer ' + GITHUB_TOKEN,
+          ...(GITHUB_TOKEN ? { Authorization: 'Bearer ' + GITHUB_TOKEN } : {}), // If we have no token and are in development, we can still make some requests.
```

Comment:
This is a somewhat complicated but I find neat way to not set any headers when we have no `GITHUB_TOKEN`. The request will go through with any headers but will not with `Authorization: NULL` header. For development purposes (see also check [above](https://github.com/jsonresume/jsonresume.org/pull/25/files#diff-5546aaab81633c80db45cdbc9c06a7155d4dae5fbe0b07b3069edc0d721ac33dR17)) it works without the `GITHUB_TOKEN` just fine. If someone hits rate limits during development, they can still set the token if they want.